### ISSUE TITLE
[Horizontal Layout] Background Color Consistency

### ISSUE THREAD

**Issue by fncolon** [zuramai/mazer#26]
## Description

Mobile Version
![Capture](https://user-images.githubusercontent.com/50759463/127768619-a3bec42d-f961-48a0-b08e-1da2857cae43.PNG)

Web Version
![Capturse](https://user-images.githubusercontent.com/50759463/127768622-f422fdbf-a5fd-4fae-9777-401d2b00ab88.PNG)

it dones responsive but the background color i think should not change too.

## Tasks

- [ ] Use `#435ebe` in `.main-navbar` background-color for mobile version instead.

**Comment by zuramai** [comment]
Yes indeed, I admit it that the navbar color on responsive mode isn't quite match, like "dead colors". I've tried the primary color but it's just not do not suit it.

Does anyone have any other color palette options?

**Comment by fncolon** [comment]
Will close this issue.
